coderd/database/queries/templateversions.sql

```diff
@@ -234,3 +225,13 @@ FROM
 WHERE
 	template_versions.id IN (archived_versions.id)
 RETURNING template_versions.id;
+
+-- name: UpdateTemplateVersionAITaskAndExternalAgentByJobID :exec
```

Comment:
Maybe `UpdateTemplateVersionFlagsByJobID`? The name might get even crazier if we add more of these post-build flag columns.